apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -198,4 +200,36 @@ export class FilesStorageUC {
 
 		return fileRecord;
 	}
+
+	async restoreFilesOfParent(userId: EntityId, params: FileRecordParams): Promise<Counted<FileRecord[]>> {
+		const [fileRecords, count] = await this.fileRecordRepo.findBySchoolIdAndParentIdAndMarkedForDelete(
+			params.schoolId,
+			params.parentId
+		);
+		if (count > 0) {
+			await this.restore(fileRecords);
+		}
+		return [fileRecords, count];
+	}
+
+	async restoreOneFile(userId: EntityId, params: SingleFileParams): Promise<FileRecord> {
+		const fileRecord = await this.fileRecordRepo.findOneByIdMarkedForDelete(params.fileRecordId);
+		await this.restore([fileRecord]);
+
+		return fileRecord;
+	}
+
+	private async restore(fileRecords: FileRecord[]) {
+		this.logger.debug({ action: 'restore', fileRecords });
+
+		await this.restoreExpires(fileRecords);
```

Comment:
We should also rename setExpires and restoreExpires bash functions in markForDelete and unmarkForDelete or?